### ISSUE TITLE
Dependency strict-event-emitter broken on 0.2.3 (set as @^2.0.0)

### ISSUE THREAD

**Issue by pedrorault** [kettanaito/msw#1186]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Browsers

Chromium (Chrome, Brave, etc.)

### Reproduction repository

https://github.com/pedrorault/msw-dependency

### Reproduction steps

npm exec tsc

### Current behavior

`node_modules/strict-event-emitter/lib/StrictEventEmitter.d.ts(13,5): error TS2416: Property 'eventNames' in type 'StrictEventEmitter<EventMap>' is not assignable to the same property in base type 'EventEmitter'.
  Type '() => (keyof EventMap)[]' is not assignable to type '() => (string | symbol)[]'.
    Type '(keyof EventMap)[]' is not assignable to type '(string | symbol)[]'.
      Type 'keyof EventMap' is not assignable to type 'string | symbol'.
        Type 'string | number | symbol' is not assignable to type 'string | symbol'.
          Type 'number' is not assignable to type 'string | symbol'.
`

### Expected behavior

Compiling without errors

**Comment by brunomarks7** [comment]
The same problem here

**Comment by kettanaito** [comment]
Hey, @pedrorault. Thanks for reporting this. 

The issue has been fixed in `strict-event-emitter@0.2.4`. Re-installing `msw` should pull that latest version of the dependency.

**Comment by nathanhannig** [comment]
getting this error on 0.2.4

```
node_modules/strict-event-emitter/lib/StrictEventEmitter.d.ts:3:43 - error TS2344: Type 'string | symbol' does not satisfy the constraint 'string'.
  Type 'symbol' is not assignable to type 'string'.

3 export declare type EventMapType = Record<string | symbol, any>;

```

Sadly not sure how to report issue to strict-event-emitter team
